src/style.css

```
@@ -1,3 +1,71 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+
+  --main-hover-color: #00ACDC;
+}
+
+h1,
+a,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+}
+
+.logo-link {
+  line-height: 0;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+}
+
+
+.nav-list-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+ .nav-link {
+   color: #000;
```

just 2 spaces
fix it in all cases
```suggestion
  color: #000;
```